apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -159,6 +169,55 @@ public ReportConfiguration() {
         dryRun = false;
         exclusionProcessor = new ExclusionProcessor();
         claimValidator = new ClaimValidator();
+        sources = new ArrayList<>();
+        reportables = new ArrayList<>();
+    }
+
+    /**
+     * Adds a file as a source of files to scan.
+     * The file must be a text file that lists files to be included.
+     * File within the file must be in linux format with a
+     * "/" file separator.
+     * @param file the file to process.
+     */
+    public void addSource(final File file) {
```

Comment:
Should we add a check here that ensures no backslashes are provided as parameter?

Reply:
ok, I just thought of it, when I read the Javadocs.